packages/fxa-shared/email/validateEmail.js

```diff
@@ -0,0 +1,32 @@
+const dns = require('dns');
+const resolver = new dns.promises.Resolver();
+const NotFoundErrorCodes = require('./emailValidatorErrors').NotFoundErrorCodes;
+
+export const results = ['MX', 'A', 'none', 'skip'].reduce(
+  (accumulator, val) => {
+    accumulator[val] = val;
+    return accumulator;
+  },
+  {}
+);
+
+const tryResolveWith = (resolveFunc) => async (domain) => {
```

Comment:
Why not write this as a standard function?

Reply:
this is just a refactor of the existing check as a temporary solution. this is how the function existed in `fxa-content-server`

i agree with your sentiment that this is poorly written but I am hesitant to rework any of the logic here rather than just pulling it out to `fxa-shared`

Reply:
Ah, got it. These two files should have the MPL added to the top.

Reply:
added